Gemfile

```diff
@@ -6,19 +6,16 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.1.4'
 
-gem 'rails', '~> 7.0.7'
+gem 'rails', '~> 7.1.2'
```

Comment:
No config change is required on this upgrade? You're bumping a minor version, usually, the rails framework has some config changes on minor upgrades, or even new stuff.

Reply:
probably did not test the app after the update  
https://github.com/Codeminer42/Punchclock/actions/runs/7133159541/job/19425240472?pr=646#step:7:18

Reply:
Tests run ok after some fixes.